tests/acceptance/features/groupMembership.feature

```diff
@@ -0,0 +1,79 @@
+@webUI @insulated @disablePreviews
+Feature: group membership
+
+As an admin
+I want to the owncloud groups to reflect the LDAP groups
+So that I only need to configure group membership once
+
+	Background:
```

Comment:
various steps here are going to fail now, and will need updating to the new core formats